scipy/special/special/cephes/lanczos.h

```diff
@@ -0,0 +1,112 @@
+/*  (C) Copyright John Maddock 2006.
+ *  Use, modification and distribution are subject to the
+ *  Boost Software License, Version 1.0. (See accompanying file
+ *  LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt)
+ */
+
+/* Both lanczos.h and lanczos.c were formed from Boost's lanczos.hpp
+ *
+ * Scipy changes:
+ * - 06-22-2016: Removed all code not related to double precision and
+ *   ported to c for use in Cephes. Note that the order of the
+ *   coefficients is reversed to match the behavior of polevl.
+ */
+
+/*
+ * Optimal values for G for each N are taken from
+ * https://web.viu.ca/pughg/phdThesis/phdThesis.pdf,
+ * as are the theoretical error bounds.
+ *
+ * Constants calculated using the method described by Godfrey
+ * https://my.fit.edu/~gabdo/gamma.txt and elaborated by Toth at
+ * https://www.rskey.org/gamma.htm using NTL::RR at 1000 bit precision.
+ */
+
+/*
+ * Lanczos Coefficients for N=13 G=6.024680040776729583740234375
+ * Max experimental error (with arbitrary precision arithmetic) 1.196214e-17
+ * Generated with compiler: Microsoft Visual C++ version 8.0 on Win32 at Mar 23 2006
```

Comment:
Yeah. This comment comes from Boost, where this was originally ported from, https://www.boost.org/doc/libs/1_84_0/boost/math/special_functions/lanczos.hpp. For each choice of N and g, you get a different set of coefficient in the expansion, and boost gives several options, noting the accuracy observed in the PhD dissertation *An Analysis of the Lanczos Gamma Approximation* by Glendon Ralph Pugh, https://web.viu.ca/pughg/phdThesis/phdThesis.pdf.

You make a good point that it would make sense to clean up the comments while translating.

Reply:
FWIW if you're not familiar with this method-I wasn't until a year or so ago-then you might have no clue where the `G, N` values come from. For this reason I think it's worthwhile to keep the comment here as written, especially because these aren't in the public api and there is no docstring to put the reference into.  Let's leave the comment here as is, for maintainenance and readability sakes.

Reply:
If you're not familiar with this method or these documents-I wasn't until a year or two ago-then it is super mysterious where these constants come from. For maintenance time and readability purposes it makes sense to keep these comments.